src/senselab/audio/workflows/health_measurements/extract_health_measurements.py

```diff
@@ -0,0 +1,93 @@
+"""This script contains a function for extracting health measurements from audio files.
+
+The initial implementation of this features extraction was started by Nicholas Cummins
+from King's College London and has since been further developed and maintained
+by the senselab community.
+"""
+
+from typing import Any, Dict, List, Optional
+
+from senselab.audio.data_structures import Audio
+from senselab.audio.tasks.features_extraction.praat_parselmouth import extract_praat_parselmouth_features_from_audios
+
+
+def extract_health_measurements(audios: List[Audio], cache_dir: Optional[str] = None) -> List[Dict[str, Any]]:
+    """Extract health measurements from audio files.
```

Comment:
`health` seems like a broad space, but perhaps this is putting a stake in the ground here :)

while many of these features have been shown to vary with health conditions, it would be good to make this more concrete with pointers and examples. this seems quite related to the phenotype vector considerations at a conceptual level. perhaps, to make this a key output of this project, we should raise the standards and have some principled way in which we add features or composite scores that have direct evidence and relevance to health. for example such principles could include high out of sample positive predictive value (precision) for a disorder/symptom tested in a large diverse dataset. i'm thinking what level of evidence would one need to include such a feature be included alongside someone's EHR.

we could also have a softer version that says these features have some evidence but do not rise to a standard that should be included in an EHR (like a blood test result) for example. perhaps this is that function and we could make a new function that says `EHR_ready_features`, which would have the higher bar.

Reply:
this is basically we are discussing in the paper with @nickcummins41. we are aware that this is just a first effort and we will need v2, v3, ... of this set. also, the details of the features inclusion and exclusion criteria will be in the paper coming with the feats. once the manuscript is ready, we can surely summarize some of the info in our doc and point to the paper for more details